end-to-end/cypress/e2e/portal/publication-page.cy.js

```diff
@@ -0,0 +1,110 @@
+// This test was written against a existing publication in the dev/test env and will need to be updated if that publication is deleted
+
+const publicationId = "cd03fa15322492f4173cdbdc30c74a62";
```

Comment:
The docker container uses the endpoints defined in `example-app.conf`. We should request that Mark's publication be added to `test` as it will be more stable and better reflect what we expect a publication to be.

Reply:
Agreed, that would be ideal; it would allow us to complete the rest of the test cases for the publication page as well - who can we discuss this with to get it done?

Reply:
I've reached out in the `feature-publicationpages` channel.